CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support of single-quoted multiline values [#450](https://github.com/dotenv-linter/dotenv-linter/pull/450) ([@DDtKey](https://github.com/DDtKey))
 
 ### 🔧 Changed
+- Fix clippy warnings [#452](https://github.com/dotenv-linter/dotenv-linter/pull/452) ([@mgrachev](https://github.com/mgrachev))
```

Comment:
Maybe it's better to call this change "Use derive (default) instead of impl"? (or smthg like this)